mmengine/dataset/dataset_wrapper.py

```diff
@@ -28,12 +28,15 @@ class ConcatDataset(_ConcatDataset):
     Args:
         datasets (Sequence[BaseDataset] or Sequence[dict]): A list of datasets
             which will be concatenated.
+        ignore_keys (List[str] or str): Ignore the keys that can be
+            unequal in `dataset.metainfo`. Defaults to None.
         lazy_init (bool, optional): Whether to load annotation during
             instantiation. Defaults to False.
     """
 
     def __init__(self,
                  datasets: Sequence[Union[BaseDataset, dict]],
+                 ignore_keys: Optional[Union[str, List[str]]] = None,
```

Comment:
Follow the google python style guide: https://google.github.io/styleguide/pyguide.html

![image](https://user-images.githubusercontent.com/57566630/195066388-f2779e9c-0565-4117-96a9-104587656a06.png)

Since ignored_keys could be `str` or `list`, its typehint should be `Union[str, List[str], None]`. Although it has not been unified in `MMEngine` :rofl: :

:rofl:
--




:rofl:
--




:rofl:
--




:rofl:
--




:rofl:
--




:rofl:
--

Reply:
😅update the typehint